programs/openbook-v2/src/lib.rs

```diff
@@ -127,6 +127,77 @@ pub mod openbook_v2 {
         Ok(None)
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub fn place_order_pegged(
+        ctx: Context<PlaceOrder>,
+        side: Side,
+
+        // The adjustment from the oracle price, in lots (quote lots per base lots).
+        // Orders on the book may be filled at oracle + adjustment (depends on order type).
+        price_offset_lots: i64,
+
+        // The limit at which the pegged order shall expire.
+        // May be -1 to denote no peg limit.
```

Comment:
for now we are enforcing not -1. so I'm assuming we should remove this and anything related to orders without peg limit